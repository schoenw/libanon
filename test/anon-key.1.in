abc123
passphrase
testing
aaa
bbbxxxccc
fvoj432holknvoe
vberbvolnjbf
4rehg54hgenbdb
fd
h
34
hehdboihvoihtsghvsdods

-
,./;'\[]
<>?:"{}

