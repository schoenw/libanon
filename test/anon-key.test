#!/bin/bash
#
# Shell script for regression testing libanon (anon-key).
#
# $Id$
#

ANON=../src/anon

RC=0
for file in anon-key.*.in; do
    $ANON key $file \
	| diff -u `basename $file .in`.out -
    if [ $? -ne 0 ]; then
 	RC=1
    fi
done

exit ${RC}