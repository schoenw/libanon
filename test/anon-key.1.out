a9993e364706816aba3e25717850c26c9cd0d89d40bd001563085fc35165329ea1ff5c5ecbdbbeef
152a97dd8e42036ae21096b9bf55456c131a7aa8deff0c25210170b91c051c03dabcbc0ae65115f5
d1c056a983786a38ca76a05cda240c7b86d7713693544000a0e6c4dd5b3dabddf512b0cb76843a8f
86f7e437faa5a7fce15d1ddcb9eaeaea377667b8e0c9035898dd52fc65c41454cec9c4d2611bfb37
23b513f6bddbe27cec268e502f730a64fe31e76c2c9bb2cf2e0f9e71697229ed76c3746af59599c4
3e5ff44349edeab3ac2d868648108feb0ed184afbe5ac2657d2a6a97367d8c983cd29e5fd920b673
f86efccb83887307d9188923045a85bf3fe30d83cb55134da59898dd64981901914252f91a650b8c
8422088cebd58c866a9058902784aa9a28b7e1a1c8a9649fc1cec79ef0edd2911e0ab3b68d3e7e62
4a0a19218e082a343a1b17e5333409af9d98f0f53c363836cf4e16666669a25da280a1865c2d2874
da39a3ee5e6b4b0d3255bfef95601890afd8070927d5482eebd075de44389774fce28c69f45c8a75
77de68daecd823babbb58edb1c8e14d7106e83bb1b6453892473a467d07372d45eb05abc2031647a
1d136a32d74e95a7ee4346f642f987241f96cd771d4d466a2ba673790252392a496ebba715ad4ddf
da39a3ee5e6b4b0d3255bfef95601890afd80709da39a3ee5e6b4b0d3255bfef95601890afd80709
da39a3ee5e6b4b0d3255bfef95601890afd807093bc15c8aae3e4124dd409035f32ea2fd6835efc9
002f5bf89528a7a6ffca1c54955a1ee494604b3f039640144f64444ce7b92169d66a3f6b4b00f939
8d80c13c2b185b88dcf9557ab5476891a3d8199845404bd0a0aba6a3f5583c444d16af882681e939
da39a3ee5e6b4b0d3255bfef95601890afd80709da39a3ee5e6b4b0d3255bfef95601890afd80709
